### DISCUSSION TITLE
Import Plex Playlist into PKC

### DISCUSSION THREAD

**Discussion by mesoller** [croneter/PlexKodiConnect#1837] [Q&A]
I recently discover Plex Meta Manager (PMM) https://github.com/meisnate12/Plex-Meta-Manager and I found this is a good combination with PKC. How can I add playlist as submenu or widget from playlist generated by PMM?

**Comment by croneter** [comment]
"Normal", non-smart Plex Playlists are synched to Kodi and become "normal" Kodi playlists. Hence have a look at the "normal" Kodi playlist features - there are no dedicated PlexKodiConnect playlist features except the synching.
